src/components/RestroPage/RestroDetails/RestroDetails.js

```diff
@@ -0,0 +1,77 @@
+import React, { Component } from "react";
+// import {defaultMemoize} from 'reselect'
+import "./RestroDetails.css"
+import listOfDishes from "./constants/restroDetails.list"
+import Menu from "./Menu"
+import {handleHelper} from './handleHelper'
+const ENUM={
```

Comment:
need not of this write listOfDishes as DISH_LIST and access it whenever required